packages/file_selector/file_selector/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
This PR is not exempt from version changes; I'm not sure why you've changed this to `NEXT` since the previous review, but that's incorrect.